testsupport/tiers/checks.go

```diff
@@ -355,6 +362,43 @@ func (a *baseextendedidlingTierChecks) GetClusterObjectChecks() []clusterObjects
 		idlers(518400, "dev", "stage"))
 }
 
+func commonToolchainLabelsNamespaceCheck() namespaceObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, _ *wait.MemberAwaitility, userName string) {
+		assertExpectedToolchainLabels(t, ns, userName)
+	}
+}
+
+func commonToolchainLabelsRoleCheck() spaceRoleObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, userName string) {
+		roles := &rbacv1.RoleList{}
+		err := memberAwait.Client.List(context.TODO(), roles, providerMatchingLabels, client.InNamespace(ns.Name))
+		require.NoError(t, err)
+		for i := range roles.Items {
+			assertExpectedToolchainLabels(t, &roles.Items[i], userName)
+		}
+	}
+}
+
+func commonToolchainLabelsRoleBindingsCheck() spaceRoleObjectsCheck {
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, userName string) {
+		roleBindings := &rbacv1.RoleBindingList{}
+		err := memberAwait.Client.List(context.TODO(), roleBindings, providerMatchingLabels, client.InNamespace(ns.Name))
+		require.NoError(t, err)
+		for i := range roleBindings.Items {
+			assertExpectedToolchainLabels(t, &roleBindings.Items[i], userName)
+		}
+	}
+}
```

Comment:
out of curiosity - why do we need this here? I thought that we already verify the labels as part of the checks for the roles and rolebindings... But I maybe missed something